lib/firmware-base/system-status.c

```diff
@@ -0,0 +1,94 @@
+#include <driver/timer.h>
+#include <freertos/FreeRTOS.h>
+#include <freertos/timers.h>
+
+#include <stdio.h>
+
+#define TIMER_DIVIDER 80  // Divide Base by 80
```

Comment:
Comment is self-evident, would you mind `#define`ing constants for an expression as to how you arrived to this value?